packages/babel-traverse/src/path/family.ts

```diff
@@ -206,36 +205,39 @@ export { get };
 export function _getKey<T extends t.Node>(
   this: NodePath<T>,
   key: string,
-  context?: TraversalContext,
+  context?: true | TraversalContext,
```

Comment:
Just a couple of questions.
Since this method is only called from `#get()` this shouldn't be marked as optional because it will default to true when no value is provided. Right?

Also this is not expected to be called from third party plugins, then it shouldn't be exported?